src/JoomlaBrowser.php

```diff
@@ -897,4 +897,38 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+    
+    /**
+     * Function to delete the menu item
+     *
+     * @param  string  $menuItem   Menu Item to be deleted
+     * @param  string  $menu    Menu type of the menu item
```

Comment:
The "joomla" code style would be, three spaces before the type, two after the longest type and two after the variable name. (3-2-2)

```php
@param   string   $a    Description
@param   boolean  $not  Description 
```